packages/intrepid2/src/Discretization/Basis/Intrepid2_NodalBasisFamily.hpp

```diff
@@ -75,6 +75,38 @@
 #include <Intrepid2_HVOL_TET_Cn_FEM.hpp>
 
 namespace Intrepid2 {
+
+  // the following defines a family of nodal basis functions for the reference Triangle
+  template<typename ExecutionSpace=Kokkos::DefaultExecutionSpace,
+           typename OutputScalar = double,
+           typename PointScalar  = double,
+           bool defineVertexFunctions = true>
+  class NodalTriangleBasisFamily
+  {
+  public:
+    // we will fill these in as we implement them
```

Comment:
I think this comment can be removed, since the whole exact sequence is filled in here.

Reply:
@CamelliaDPG Thanks, I removed that comment.